db/migrate/20220403123845_create_bsnl_delivery_details.rb

```diff
@@ -0,0 +1,14 @@
+class CreateBsnlDeliveryDetails < ActiveRecord::Migration[5.2]
+  def change
+    create_table :bsnl_delivery_details do |t|
+      t.string :message_id
+      t.string :message_status
+      t.string :result
```

Comment:
If we wanna keep the PRs a bit scoped, wanna add just the fields we're using into this PR, and introduce the rest just-in-time? AKA the PR that populates them via API or consumes them somewhere?

Reply:
That PR is coming right next up, so I'd like to keep it into a single migration for ease.

Reply:
Makes sense if we have a very good picture of how this shakes out 👍